Mathlib/GroupTheory/Subsemigroup/Center.lean

```diff
@@ -164,12 +278,12 @@ namespace Subsemigroup
 
 section
 
-variable (M) [Semigroup M]
```

Comment:
I think things would be clearer with a separate `section Semigroup` and `section Mul`. Same in all the other `Center` files.

Reply:
Fair enough. Done.